hbase_sql/src/main/java/com/splicemachine/derby/impl/SpliceSpark.java

```diff
@@ -108,11 +111,30 @@ public static synchronized SparkSession getSessionUnsafe() {
             } catch (IOException e) {
                 throw new RuntimeException(e);
             }
-            System.exit(0);
-            session = initializeSparkSession();
+            if (isOlapWorker)
+                System.exit(0);
+            sessionToUse = session = initializeSparkSession();
             ctx =  new JavaSparkContext(session.sparkContext());
         }
-        return session;
+        else {
+            if (sessionToUse == null || needsReinitialization) {
+                String allowMultipleContextsString = null;
+                try {
+                    allowMultipleContextsString = session.conf().get("spark.driver.allowMultipleContexts");
```

Comment:
Can we get rid of spark.driver.allowMultipleContexts in tests and remove this special case?

Reply:
Done.  spark.driver.allowMultipleContexts doesn't exist in the tests.  This is just a spark setting I was experimenting with.  I've removed this option.